src/deploy-cromwell-on-azure/scripts/cromwellazure.service

```diff
@@ -1,6 +1,6 @@
 [Unit]
 Description=CromwellOnAzure
-RequiresMountsFor=/data
+RequiresMountsFor=/data /mnt
```

Comment:
This forces the system to process all the mounts listed in /etc/fstab before starting startup.sh. Since startup.sh unmounts all, recreates fstab entries, and mounts them all, we don't want the system mounting entries from fstab at the same time that startup.sh is running.